styleguide/_themes/derek/scss/globalElements/mainnav.scss

```diff
@@ -344,20 +344,19 @@
     width: 95px;
   }
 
-  .main-nav {
-
-    .navbar-phoneicon,
-    .navbar-chaticon,
-    .navbar-emailicon,
-    .navbar-searchicon {
-      @include make-xs-column(2);
-      border-left: solid 1px $gray-darker;
-      height: 63px;
```

Comment:
this was unnecessary code that I didn't clean out when sexy nav was being developed.